src/implementations/vanilla/src/components/global-nav/side-nav/search/search.js

```diff
@@ -47,6 +53,24 @@ class Search extends Core {
   showClearIcon() {
     this._findDOMEl('.hig__global-nav__side-nav__search__clear', this.el).classList.add('hig__global-nav__side-nav__search__clear--show');
   }
+
+  _setIcons(){
+    const mountSearchIcon = this._findDOMEl('.hig__global-nav__side-nav__search__icon', this.el);
+    this._findOrCreateIconComponent(mountSearchIcon, 'search').setNameOrSVG("search-small");
+
+    const mountClearIcon = this._findDOMEl(".hig__global-nav__side-nav__search__clear", this.el);
+    this._findOrCreateIconComponent(mountClearIcon, 'clear').setNameOrSVG("close-small");
+  }
+
+  _findOrCreateIconComponent(mountElOrSelector, name = "icon") {
```

Comment:
I think it's worth considering mixing in this method from an *HasIcon* type of module

Reply:
oh good catch. I meant to fix the eye icon.  and yeah sean and I talked about creating a mixin. It's just so small. But I can create a mixin for that function